terminology/common-terminology.md

```diff
@@ -4,13 +4,13 @@
 
 As more companies and open source projects begin to use similar terms to describe similar paradigms 	that build upon hardware-based Trusted Execution Environments (TEEs), it will be increasingly important that vendors use consistent terminology that describes the ways in which these new capabilities are applied within different functional domains. 
 
-The Confidential Computing Consortium has defined [^1] confidential computing as “the protection of data in use by performing computation in a hardware-based Trusted Execution Environment”, and identified three primary attributes for what constitutes a Trusted Execution Environment: data integrity, data confidentiality, and code integrity. As described in Confidential Computing: Hardware-Based Trusted Execution for Applications and Data, four additional attributes may be present (code confidentiality, programmability, recoverability, and attestability) but are not strictly necessary for a computational environment to be classified as confidential computing. 
+The Confidential Computing Consortium has defined [^1] Confidential Computing as “the protection of data in use by performing computation in a hardware-based Trusted Execution Environment”, and identified three primary attributes for what constitutes a Trusted Execution Environment: data integrity, data confidentiality and code integrity. As described in Confidential Computing: Hardware-Based Trusted Execution for Applications and Data, four additional attributes may be present (code confidentiality, programmability, recoverability, and attestability) but are not strictly necessary for a computational environment to be classified as Confidential Computing. 
```

Comment:
Since I continue to hear feedback that people want attestability to be a requirement, it may be worth revisiting whether we can get consensus on that, especially given the attestation section at the bottom of this doc which woudl seem to be much stronger if we did declare that attestability was required for confidential computing (even if it's not required in the definition of a TEE).  E.g., I got one request to consider changing "in a hardware-based TEE" to "in a hardware-attested TEE" in order to work in the attestation requirement.  We can discuss in the meeting.

Reply:
I get the above, but I think it misses the point.  "Hardware-based TEE" is about the runtime and execution, and is completely orthogonal to the attestation.  What's more, the attestation isn't hardware-based: the attestation _measurement_ is hardware-based.  The validation of the attestation is what completes the attestation process, and validates the status of the TEE instance.